rest-api/dao/participant_dao.py

```diff
@@ -263,6 +244,8 @@ def to_client_json(self, model):
     format_json_date(client_json, 'withdrawalTime')
     format_json_date(client_json, 'suspensionTime')
     client_json['awardee'] = client_json['hpoId']
+    if 'siteId' in client_json:
```

Comment:
Should format_json_site do this?

Reply:
it does if site_id is not None, lots of fields get called on format_json_site, I ran into a circumstance where I could still have SiteId here and would return null on client side, this was just to catch that IF it happens.